### ISSUE TITLE
Searching for posts doesn’t work

### ISSUE THREAD

**Issue by thatpaulallen** [Balackburn/Apollo#32]
It loads forever and than says No Results. I’m on an iPhone 15 Pro, side loaded with Sidestore

**Comment by Balackburn** [comment]
> It loads forever and than says No Results. I’m on an iPhone 15 Pro, side loaded with Sidestore

Have you defined an API key?

**Comment by thatpaulallen** [comment]
Yep. I’ve been using the app with no issues since last year. This problem just cropped up. Today is the first time I noticed it
